pymeasure/instruments/temptronic/temptronic_base.py

```diff
@@ -0,0 +1,826 @@
+        """,
+        )
+
+
+    auxiliary_condition_code = Instrument.measurement(
+        "AUXC?",
+        """Read the auxiliary condition register.
+
+        bit 10 – reserved
+        bit 9 – ramp mode
+        bit 8 – Manual Mode= 1, Program= 0
+        bit 7 – reserved
+        bit 6 – ready for operation = 1, startup sequence = 0
+        bit 5 – flow on = 1, flow off = 0
+        bit 4 – DUT mode = 1, air-control mode = 0
+        bit 3 – heat only mode = 1, compressor on =0
+        bit 2 – head up = 1, head down = 0
+        bit 1 – reserved
+        bit 0 – reserved
+        """,
+        validator=truncated_range,
+        values=[0, 1023]
+        )
+
+    copy_active_setup_file = Instrument.setting(
+        "CFIL %g",
+        """Local lockout – the System touch screen controls are disabled, and
+        no “Return to local” button appears on the panel.
+        """,
+        validator=strict_range,
+        values=[1, 12]
+        )
+
+    clear_errors = Instrument.setting(
+        "CLER",
+        """Local lockout – the System touch screen controls are disabled, and
+        no “Return to local” button appears on the panel.
+        """
+        )
+
+    compressor_enable = Instrument.setting(
+        "COOL %g",
+        """Turn the compressor on or off.
+        compressor_enable = True (turn the compressor on)
+        compressor_enable = False (turn the compressor off)
+        """,
+        validator=strict_discrete_set,
+        values={0,1,True,False}
+        )
+
+    total_cycle_count = Instrument.control(
+        "CYCC?", "CYCC %g",
+        """Set the cycle count.
+
+        CYCC nnnn -- where nnnn is the number (1 - 9999) of cycles to do.
+        Read the number of cycles to do.
+
+        total_cycle_count = nnnn (start cycling)
+        total_cycle_count = 0    (stop cycling)
+        NOTE: When all cycles have been completed or when cycling was
+        stopped on failure, it is necessary to send a CYCL 0 command to
+        reset the system.
+        """,
+        validator=truncated_range,
+        values=[0, 9999]
+        )
+
+    cycling_enable = Instrument.setting(
+        "CYCL %g",
+        """CYCL Start/stop cycling.
+        cycling_enable = True  (start cycling)
+        cycling_enable = False (stop cycling)
+        NOTE: When all cycles have been completed or when cycling was
+        stopped on failure, it is necessary to send a CYCL 0 command to
+        reset the system.
+        Read the cycle number (current value if cycling, last value if not).
+        NOTE: If Version 1 compatibility mode is enabled, CYCL? returns
+        the number of fully completed cycles.
+        """,
+        validator=strict_discrete_set,
+        values={0,1,True,False}
+        )
+
+    current_cycle_count = Instrument.measurement(
+        "CYCL?",
+        """Read the system’s current draw (amperage).
+        """,
+        )
+
+    error_code = Instrument.measurement(
+        "EROR?",
```

Comment:
Is there a typo here, or is the instrument command really EROR, with 1 R?

Reply:
yes, no typo, that's correct, I verified again and put in a comment.

Reply:
funny :D
thanks!